src/repository/CheckForUpdateRepo.ts

```diff
@@ -0,0 +1,50 @@
+import { ExtensionContext, Memento } from "vscode";
+import { ConfigurationTarget } from "../ConfigurationTarget";
+
+type LastUpdated = {
```

Comment:
I think it's more common to use interface for such data types

Reply:
I don't have a strong opinion, but I prefer to use `type` over `interface` for the data types that shouldn't be extended (by inheritance or declaration merging).